include/rtc/configuration.hpp

```diff
@@ -77,6 +77,7 @@ struct RTC_CPP_EXPORT Configuration {
 	bool disableAutoNegotiation = false;
 	bool disableAutoGathering = false;
 	bool forceMediaTransport = false;
+	bool skipCheckFingerprint = false;
```

Comment:
You could call the flag `disableFingerprintVerification` for consistency with `WebSocketConfiguration`, otherwise it looks good.

Reply:
make sense.